### PR TITLE
Windows production mode command

### PR DESCRIPTION
**Pull request by PeterYusuke**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### Description
Add production mode command for Windows PC.
We can not use Gunicorn on Windows now, so I add Uvicorn command.
I kept the code for the Linux-based machine because it's not better to use only Uvicorn than Gunicorn.

### CODE REVIEW

**Comment by PeterYusuke** [comment]
Close #515.

File: pynecone/utils.py [hunk 1]
```diff
@@ -799,17 +799,31 @@ def run_backend_prod(
...
-        f"0.0.0.0:{port}",
-        "--workers",
-        str(num_workers),
-        "--threads",
-        str(num_workers),
-        "--log-level",
-        str(loglevel),
-        f"{app_name}:{constants.APP_VAR}()",
-    ]
+    command = []
```
Comment (picklelo) [inline pynecone/utils.py:802]: Don't need to define the variable here, we can just do it within the if/else statements

File: pynecone/utils.py [hunk 2]
```diff
@@ -799,17 +799,31 @@ def run_backend_prod(
...
+    if platform.system() == "Windows":
+        command += constants.RUN_BACKEND_PROD_WINDOWS + [
```
Comment (picklelo) [inline pynecone/utils.py:804]: These can just be `command =` instead of `+=`

File: pynecone/utils.py [hunk 3]
```diff
@@ -799,17 +799,31 @@ def run_backend_prod(
...
+            "--port",
+            str(port),
+            "--workers",
+            str(num_workers),
+            "--log-level",
+            loglevel,
+            f"{app_name}:{constants.APP_VAR}",
+        ]
+    else:
+        command += constants.RUN_BACKEND_PROD + [
```
Comment (picklelo) [inline pynecone/utils.py:816]: Here also

**Review by picklelo — COMMENTED** [review]
Thanks for the update! Just a small comment

File: pynecone/utils.py (hunk 3 above)
Comment (picklelo) [inline pynecone/utils.py:816]: It seems like all the args we pass in are the same. Can we then change this to just
```python
command = [constants..RUN_BACKEND_PROD_WINDOWS] if platform.system() == "Windows" else [constants.RUN_BACKEND_PROD]
command += [
    # Common flags here
]

```
If there are some special flags, we can put them in the if/else but move the common ones out.

Comment (PeterYusuke) [reply pynecone/utils.py:816]: OK. Only workers arg is the same, others are little bit different.
I will make common arg list.

**Comment by PeterYusuke** [comment]
Changed command list. Please check.

**Review by picklelo — APPROVED** [review]
nice

**Pull request merged**